arc/benchmark.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Arc ClickBench Complete Benchmark Script
+# This script installs Arc, loads data, and runs the benchmark
+
+set -e
+
+# Install Python and dependencies
+echo "Installing dependencies..."
+pip3 install duckdb requests
+
+# Download and prepare dataset
+echo "Downloading ClickBench dataset..."
+wget --continue 'https://datasets.clickhouse.com/hits_compatible/hits.parquet'
```

Comment:
Please add `--progress=dot:giga` (similar to what other benchmark scripts use as command line option to `wget`).